pkg/didcomm/dispatcher/api.go

```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
 )
 
-// Service protocol service
-type Service interface {
+// ProtocolService is service interface for protocol services
+type ProtocolService interface {
```

Comment:
A short description about difference between protocol services v message services would be helpful.

Reply:
done